pkg/sql/execinfrapb/data.proto

```diff
@@ -329,6 +329,8 @@ message RemoteProducerMetadata {
       (gogoproto.customname) = "FlowID",
       (gogoproto.customtype) = "FlowID"];
     optional bool drained = 9 [(gogoproto.nullable) = false];
+		// ProcessorID is the ID of the processor that published the metadata.
```

Comment:
nit / (why isn't there a formatter for this 😢 ): this comment is indented with tabs when it should be spaces

Reply:
i can't tell you how many times i've tried to coax vscode to use spaces instead of tabs.